lessons/module-3/react-2-the-how.md

```diff
@@ -185,62 +175,55 @@ Let's figure out what should be a component in our app.
 
 Because we want App to hold onto our list of ideas, let's think about how we're going to store that information.
 
-In our first React lesson, we learned about _state_. In order to use component state, we need our component to be a class.
+In our first React lesson, we learned about _state_. In order to use component state, we need to import useState from React.
```

Comment:
Might be good to clarify this by saying "we need to import the useState hook from React."